dist/images/ovnkube.sh

```diff
@@ -496,11 +511,12 @@ ovn-controller () {
   echo "=============== ovn-controller - (wait for ovs)"
   wait_for_ovs
 
-  echo "=============== ovn-controller - (wait for northdb)"
-  wait_for_northdb
+  echo "=============== ovn-controller - (wait for northd)"
+  wait_for_northd
```

Comment:
Turns out (per discussion internally) that ovn-controller doesn't need to wait for northd.  We can just start it up, and it'll spin waiting to connect to the SB database until ovnkube node tells it what to do.